template-engine/chapter.md

```diff
@@ -409,7 +409,8 @@ Expressions in `{{ ... }}` are computed, converted to strings, and added to the
 result.  Dots in the expression are handled by the `do_dots` function passed
 into our function, because the meaning of the dotted expressions depends on the
 data in the context: it could be attribute access or item access, and it could
-be a callable.
+be a callable. ((FIXME: Query "a callable" -- would "a callable function" be better?
+Is "callable" commonly understood to be used as a noun? --ARB ))
```

Comment:
"callable" is a term from Python, and can include functions, methods, classes, or even specially constructed objects.